tools/extra/fpgabist/dma/fpga_dma.h

```diff
@@ -47,19 +47,17 @@ extern "C" {
 * or memory-mapped. Streaming interfaces are not currently
 * supported.
 */
-typedef enum {
-	HOST_TO_FPGA_MM = 0, //Memory mapped FPGA interface
-	FPGA_TO_HOST_MM, //Memory mapped FPGA interface
-	FPGA_TO_FPGA_MM, //Memory mapped FPGA interface
-	FPGA_MAX_TRANSFER_TYPE,
-} fpga_dma_transfer_t;
-
-typedef struct _dma_handle_t *fpga_dma_handle;
+	typedef enum {
+		HOST_TO_FPGA_MM = 0,	//Memory mapped FPGA interface
+		FPGA_TO_HOST_MM,	//Memory mapped FPGA interface
+		FPGA_TO_FPGA_MM,	//Memory mapped FPGA interface
+		FPGA_MAX_TRANSFER_TYPE,
+	} fpga_dma_transfer_t;
 
+	typedef struct _dma_handle_t *fpga_dma_handle;
```

Comment:
Indenting seems to be off

Reply:
@asgibbons can you respond to this inquiry?

Reply:
These files are licensed under BSD.  I got confirmation from Ru on 3/19/18.  See attached email.

Subject	RE: DMA Recommendations (was: RE: Let's Make a Call on DMA)
From	Pan, Ru
To	Gibbons, Scott
Cc	Rao, Shiva
Sent	Monday, March 19, 2018 2:01 PM

Change typo:
 
OTC expert
 
From: Pan, Ru 
Sent: Monday, March 19, 2018 2:01 PM
To: Gibbons, Scott <scott.gibbons@intel.com>
Cc: Rao, Shiva <shiva.rao@intel.com>
Subject: RE: DMA Recommendations (was: RE: Let's Make a Call on DMA)
 
Hi, Scott:
 
I got confirmation from OTC exporter.  You can redistribute this file with OPAE release (BSD license).
 
 
From: Pan, Ru 
Sent: Sunday, March 18, 2018 9:17 PM
To: Gibbons, Scott <scott.gibbons@intel.com>
Cc: Rao, Shiva <shiva.rao@intel.com>
Subject: RE: DMA Recommendations (was: RE: Let's Make a Call on DMA)
 
Hi, Scott:
 
Please make sure this file is available from Intel white list.
https://whitelisttool.amr.corp.intel.com/
 
The license looks OK to me.  Will confirm with OTC.
 
From: Gibbons, Scott 
Sent: Saturday, March 17, 2018 6:49 AM
To: Pan, Ru <ru.pan@intel.com>
Cc: Rao, Shiva <shiva.rao@intel.com>
Subject: RE: DMA Recommendations (was: RE: Let's Make a Call on DMA)
 
Hi, Ru.  I need to add a file from an open-source package into OPAE for DMA and need to get it added to our IP plan.  The file is x86-sse2.S and comes from the tinymembench project.  The license reads:
 
/*	
	* Copyright © 2011-2016 Siarhei Siamashka <siarhei.siamashka@gmail.com>
	*
	* Permission is hereby granted, free of charge, to any person obtaining a
	* copy of this software and associated documentation files (the "Software"),
	* to deal in the Software without restriction, including without limitation
	* the rights to use, copy, modify, merge, publish, distribute, sublicense,
	* and/or sell copies of the Software, and to permit persons to whom the
	* Software is furnished to do so, subject to the following conditions:
	*
	* The above copyright notice and this permission notice (including the next
	* paragraph) shall be included in all copies or substantial portions of the
	* Software.
	*
	* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
	* IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
	* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
	* THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
	* LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
	* FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
	* DEALINGS IN THE SOFTWARE.
	*/

Reply:
@luebbers why do you say indenting is off? It passes CI tests?
Are you saying it should start at the beginning of the line?

Reply:
Yeah, why aren't the prototypes at the beginning of the line?